### PR TITLE
Feat: LOGINPAGE HTML

### PR DESCRIPTION
**Pull request by JeckJonnyQ**
### What type of PR is this?

- [x] 🌟 Feature
- [x] 🛠 Bug Fix
- [x] ⚙️ Refactor
- [ ] 📖 Documentation

### Description

- **Brief Overview:**
Added login page and login form.

### Additional Information

- **Screenshots/Links:**
 image

### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [ ] 📝 I have commented my code, particularly in hard-to-understand areas.
- [ ] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.

### CODE REVIEW

File: src/components/FormWrapper/FormWrapper.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ function FormWrapper({
       break;
     case "Login":
       className = "login-page";
-      description = "Enter your credentials to log in.";
+      description = "Enter your username and password to login.";
```
Comment (AndreiMakhnach2209) [inline src/components/FormWrapper/FormWrapper.tsx]: Not quite like that. We need an `email` and a `password`.

**Review by AndreiMakhnach2209 — APPROVED** [review]

File: src/components/FormWrapper/FormWrapper.tsx (hunk 1 above)
Comment (JeckJonnyQ) [reply src/components/FormWrapper/FormWrapper.tsx]: Completed

File: src/router.tsx [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import NotFound from "pages/NotFound/NotFound";
 import { RouteObject, Navigate } from "react-router-dom";
 
+import LoginPage from "./pages/LoginPage/LoginPage";
```
Comment (Oleg-Melnikow) [inline src/router.tsx]: please use import like first line) in webpack used alias for it

File: src/pages/LoginPage/LoginPage.scss [hunk 3]
```diff
@@ -0,0 +1,37 @@
+.login-page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  max-width: 100%;
+  width: 100%;
+  min-height: 100vh;
+
+  .login-page__content {
```
Comment (Oleg-Melnikow) [inline src/pages/LoginPage/LoginPage.scss:9]: can we use class name in this case  like this `&__content`

**Review by Oleg-Melnikow — COMMENTED** [review]
Please, check my comments in your code.

File: src/router.tsx (hunk 2 above)
Comment (JeckJonnyQ) [reply src/router.tsx]: Completed

**Review by Oleg-Melnikow — APPROVED** [review]
Good job!

File: src/pages/LoginPage/LoginPage.scss (hunk 3 above)
Comment (JeckJonnyQ) [reply src/pages/LoginPage/LoginPage.scss:9]: I haven’t edited this comment yet, I’ll correct it in the next PR

**Pull request merged**